iOS/Flows/Conversation/ConversationListCollectionViewLayout.swift

```diff
@@ -123,13 +123,6 @@ class ConversationListCollectionViewLayout: UICollectionViewFlowLayout {
         return CGPoint(x: newXOffset, y: proposedContentOffset.y)
     }
 
-    override func targetContentOffset(forProposedContentOffset proposedContentOffset: CGPoint) -> CGPoint {
-        self.sendCenterUpdateEventIfNeeded(withContentOffset: proposedContentOffset)
-
-        return self.targetContentOffset(forProposedContentOffset: proposedContentOffset,
-                                        withScrollingVelocity: .zero)
-    }
-
```

Comment:
Removing this fixed a lot of the issues with the conversation not being centered. 
It seems that overriding this causes weird issues when the collection view both scrolling and animating a layout change.